staging/src/k8s.io/api/flowcontrol/v1beta1/types.go

```diff
@@ -57,6 +57,11 @@ const (
 	ResponseHeaderMatchedFlowSchemaUID                 = "X-Kubernetes-PF-FlowSchema-UID"
 )
 
+const (
+	// AutoUpdateAnnotationKey is the name of an annotation which prevents reconciliation if set to "false"
```

Comment:
This comment needs to be expanded a lot, the semantics of this PR need to be explained, the reader is unlikely to get them all from just the word "reconciliation".

Reply:
yes, expanded the comment, please check